contrib/babelfishpg_common/src/collation.c

```diff
@@ -119,6 +119,34 @@ coll_translate_t coll_translations[] =
 };
 #define TOTAL_COLL_TRANSLATION_COUNT (sizeof(coll_translations)/sizeof(coll_translations[0]))
 
+coll_translate_t coll_translations_tsql[] =
+{
+	{ "bbf_unicode_bin2", "latin1_general_bin2", 1252 },
```

Comment:
It makes sense to have this just to make it working with the other servers, but this is against our tenet/policy. The expectation from customer POV is that we get the exact DDL which user used for creation. We our not being transparent here, this should be approved by Senior DBEs once

Reply:
Talked to Rob about it, we can go with this approach. We need to document this properly.